templates/montreal/2_wong.txt

```diff
@@ -0,0 +1,54 @@
+TITLE: Conseillère de la ville / City Council Chair Cathy Wong
+EMAIL: cathy.wong@ville.montreal.qc.ca
```

Comment:
This might be a duplicate of `1_plante.ext`

Reply:
thanks for that! I updated 1_plante.txt